layouts/partials/ecosystem/registry/entry.html

```diff
@@ -105,7 +109,7 @@ <h4 class="card-title mb-0 d-flex flex-row">
             <span class="badge rounded-pill text-bg-secondary text-white" title="This package is used in the OpenTelemetry Demo!"><i class="fa-solid fa-shapes"></i> OTel Demo</span>
             {{ end -}}
             {{ if $deprecated -}}
-            <span class="badge rounded-pill text-bg-danger text-white" title=""><i class="fa-solid fa-ban"></i> deprecated</span>
+            <span class="badge rounded-pill text-bg-danger text-white" title="This package is deprecated."><i class="fa-solid fa-ban"></i> deprecated</span>
```

Comment:
Add a few more words what deprecated means, for example that it is recommended to no longer use it or if it is in use right now to look for a replacement.